### ISSUE TITLE
I can't compile, persistent error

### ISSUE THREAD

**Issue by ShinobiMega** [fernandoigor/BRWheel#6]
hello, i did all the configuration methods to compile, i was not successful, could someone say the project is really working?

**Comment by fernandoigor** [comment]
yes, but you need to use an old version of arduino, arduino-1.0.5
________________________________
De: lirogames ***@***.***>
Enviado: sexta-feira, 23 de julho de 2021 13:07
Para: fernandoigor/BRWheel ***@***.***>
Cc: Subscribed ***@***.***>
Assunto: [fernandoigor/BRWheel] I can't compile, persistent error (#6)


hello, i did all the configuration methods to compile, i was not successful, could someone say the project is really working?

**Comment by ShinobiMega** [comment]
hello, thanks for your attention. I realized this and installed version 1.0.5r2 is it? just found is.
I tried to compile and now only 1 error appears, but I couldn't find it.
is it the arduino version yet?
I'm trying to compile with arduino and not with micro visuals.

![image](https://user-images.githubusercontent.com/7413750/126879553-93592f50-9c30-4f81-a1fc-cf732a6c34ed.png)


Error:

brWheel.cpp.o: In function `loop':
C:\Program Files (x86)\Arduino\arduino1.0.5-r2/brWheel.ino:189: undefined reference to `Joystick_::send_16_10_10_10(int, unsigned int, unsigned int, unsigned int, unsigned int)'

![image](https://user-images.githubusercontent.com/7413750/126879569-a0367bd1-d454-4290-9457-93ce69a19a69.png)

**Comment by ShinobiMega** [comment]
I realized that you are Brazilian, how fantastic I am too. Hahaha.
would be able to help me in this source, I'm thinking that someone changed the files, only the problem in this line.
if I put // in front of the line
SendInputReport((s16)turn, (u16)accel, (u16)brake, (u16)clutch, buttons);

I can compile, but apparently it won't read the external values.
I looked all I could, it is declared its change to 4 pot + buttons.
I really didn't understand...

I took the file you posted esp wheel and made the compilation, and it was the first time following the steps of the library files and such, I did the same procedure that is in your .md and this error always appears.

if you want, you can send an email or call my whats, thank you for your attention and patience.
thank you and see you later.

**Comment by ShinobiMega** [comment]
oops, I tried to compile by Visual studio 2017 using mv more problem appeared.

Compiling 'brWheel' for 'Arduino Leonardo'
 
brWheel.ino: In function void setup()
 
brWheel.ino: 196: error: 'InitInputs' was not declared in this scope
 
brWheel.ino: 205: error: 'InitPWM' was not declared in this scope
 
brWheel.ino: 206: error: 'setPWM' was not declared in this scope
 
brWheel.ino: In function void loop()
 
brWheel.ino: 219: error: 'ReadAnalogInputs' was not declared in this scope
 
brWheel.ino: 242: error: 'setPWMDir' was not declared in this scope
 
brWheel.ino: 248: error: 'nextInputState' was not declared in this scope
 
brWheel.ino: 253: error: 'AverageAnalogInputs' was not declared in this scope
 
brWheel.ino: 259: error: 'readInputButtons' was not declared in this scope
 
brWheel.ino: 262: error: 'ClearAnalogInputs' was not declared in this scope
 
brWheel.ino: 264: error: 'readSerial' was not declared in this scope
 
Inputs.ino: In function void InitInputs()
 
Inputs.ino: 137: error: 'InitShiftRegister' was not declared in this scope
ffb.ino: At global scope
 
ffb.ino: 45: error: 'FfbproEnableInterrupts' was not declared in this scope
 
ffb.ino: 46: error: 'FfbproGetSysExHeader' was not declared in this scope
ffb_pro.ino: In member function void BRFFB::calibrate()
 
ffb_pro.ino: 272: error: 'setPWMDir' was not declared in this scope
Error compiling project sources
Build failed for project 'brWheel'
